maestro-app/src/main/java/bio/overture/maestro/app/infra/adapter/outbound/indexing/elasticsearch/AnalysisCentricElasticSearchAdapter.java

```diff
@@ -240,4 +241,13 @@ private void deleteByAnalysisIdRunnable(@NonNull String analysisId) {
         QueryBuilders.boolQuery().must(QueryBuilders.termQuery("analysis_id", analysisId)));
     this.elasticsearchRestClient.deleteByQuery(deleteByQueryRequest, RequestOptions.DEFAULT);
   }
+
+  private Map<String, Object> scriptedDocument(
+      ObjectMapper mapper, AnalysisCentricDocument originalDocument) {
+    val newMapDoc = mapper.convertValue(originalDocument, Map.class);
+
+    // remove repositories from Map as we are not updating repositories
+    newMapDoc.remove("repositories");
+    return newMapDoc;
+  }
```

Comment:
I like this solution, just think we could use some more comments for future reference. Specifically why we are not updating repositories, maybe just need to say that the Update method generates the repositories property based on the currently stored document value of repositories.